contracts/interfaces/IAxelarGasService.sol

```diff
@@ -84,6 +84,8 @@ interface IAxelarGasService is IUpgradable {
 
     event NativeExpressGasAdded(bytes32 indexed txHash, uint256 indexed logIndex, uint256 gasFeeAmount, address refundAddress);
 
+    event Refunded(bytes32 indexed txHash, uint256 indexed logIndex, address payable receiver, address token, uint256 amount);
```

Comment:
This differs from the precedent established by the other events - those suggest an approach like this:

```
event NativeGasRefunded(bytes32 indexed txHash, uint256 indexed logIndex, address payable receiver, uint256 amount);

event GasRefunded(bytes32 indexed txHash, uint256 indexed logIndex, address payable receiver, address token, uint256 amount); 
```

A single event is pragmatic though, since it's not difficult to check for `address(0)` to figure out what token was refunded. No problem from a consumer POV, just thought I'd point it out

Reply:
Since the method used is the same for either case, I think we should use the same event